onnx/test/hub_test.py

```diff
@@ -105,6 +105,15 @@ def test_download_model_with_test_data(self) -> None:
             member="test_data_set_0", container=files, msg="Test data not found"
         )
 
+    def test_model_with_preprocessing(self) -> None:
+        # TODO(janton): remove preprocessing_repo once https://github.com/onnx/models/pull/594 is merged
+        model = hub.load_composite_model(
+            "ResNet50-fp32",
+            preprocessing_model="ResNet-preproc",
+            preprocessing_repo="jantonguirao/models:resnet_preproc",
```

Comment:
Is this test relying on o external data?

Reply:
I just merged the PR in onnx model zoo and now we can use that one as the test here. Thanks!

Reply:
Thanks, I updated the test